test/api_test.js

```diff
@@ -100,4 +116,9 @@ suite('api', () => {
     assert.equal(builds.builds[0].repository, 'xyz');
     assert.equal(builds.builds[0].sha, 'y650871208002a13ba35cf232c0e30d2c3d64783');
   });
+
+  test('integration installation', async function() {
+    let result = await helper.github.isRepoSet('qwerty', 'coolRepo');
+    assert.deepEqual(result, {installed: true});
+  });
```

Comment:
Yeah, I planned those... I have these test cases: "Creating an entity correctly", "Returning true for existing repo", "Returning false for absent repo", "Returning false for absent owner", "Notifying user about invalid input". Are those good? Anything else to add?